UnitTests/AppAuthTV/OIDTVAuthorizationResponseTests.m

```diff
@@ -0,0 +1,285 @@
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+ */
+
+#import "OIDTVAuthorizationResponseTests.h"
+
+#if SWIFT_PACKAGE
+@import AppAuthTV;
+#else
+#import "Source/AppAuthCore/OIDScopeUtilities.h"
+#import "Source/AppAuthCore/OIDURLQueryComponent.h"
+#import "Source/AppAuthTV/OIDTVAuthorizationRequest.h"
+#import "Source/AppAuthTV/OIDTVAuthorizationResponse.h"
+#import "Source/AppAuthTV/OIDTVServiceConfiguration.h"
+#import "Source/AppAuthTV/OIDTVTokenRequest.h"
+#endif
+
+/*! @brief Test value for the @c TVAuthorizationEndpoint property.
+ */
+static NSString *const kTestTVAuthorizationEndpoint = @"https://www.example.com/device/code";
+
+/*! @brief Test value for the @c tokenEndpoint property.
+ */
+static NSString *const kTestTokenEndpoint = @"https://www.example.com/token";
+
+/*! @brief Test key for the @c additionalParameters property.
+ */
+static NSString *const kTestAdditionalParameterKey = @"A";
+
+/*! @brief Test value for the @c additionalParameters property.
+ */
+static NSString *const kTestAdditionalParameterValue = @"1";
+
+/*! @brief Test value for the @c clientID property.
+ */
+static NSString *const kTestClientID = @"ClientID";
+
+/*! @brief Test value for the @c clientSecret property.
+ */
+static NSString *const kTestClientSecret = @"ClientSecret";
+
+/*! @brief Key for the @c verificationURI property.
+ */
+static NSString *const kVerificationURIKey = @"verification_uri";
+
+/*! @brief Alternative key for the @c verificationURI property. If "verification_uri" is not found
+        in the response, a "verification_url" key is considered equivalent.
+ */
+static NSString *const kVerificationURIAlternativeKey = @"verification_url";
+
+/*! @brief Test value for the @c verificationURI property.
+ */
+static NSString *const kTestVerificationURI = @"https://www.example.com/device";
+
+/*! @brief Key for the @c verificationURIComplete property.
+ */
+static NSString *const kVerificationURICompleteKey = @"verification_uri_complete";
+
+/*! @brief Test value for the @c verificationURIComplete property.
+ */
+static NSString *const kTestVerificationURIComplete = @"https://www.example.com/device/UserCode";
+
+/*! @brief Key for the @c userCode property.
+ */
+static NSString *const kUserCodeKey = @"user_code";
+
+/*! @brief Test value for the @c userCode property.
+ */
+static NSString *const kTestUserCode = @"UserCode";
+
+/*! @brief Key for the @c deviceCode property.
+ */
+static NSString *const kDeviceCodeKey = @"device_code";
+
+/*! @brief Test value for the @c deviceCode property.
+ */
+static NSString *const kTestDeviceCode = @"DeviceCode";
+
+/*! @brief Key for the @c expirationDate property.
+ */
+static NSString *const kExpiresInKey = @"expires_in";
+
+/*! @brief Test lifetime value used for the @c expirationDate property.
+ */
+static long long const kTestExpiresIn = 1800;
```

Comment:
Since the date is interpreted as a long long, I was going to store it as such, although an int would work. Should I change it to int? https://github.com/openid/AppAuth-iOS/blob/dev-tvos/Source/AppAuthCore/OIDAuthorizationResponse.m#L99

Reply:
I got the long long designation actually from OIDAuthorizationResponseTests

Reply:
It's been a while now, but I think the reason we use `longLongValue` is that `longValue` isn't [an option](https://developer.apple.com/documentation/foundation/nsstring/1417731-longlongvalue), and that `int` isn't generally used for timestamps (although I agree it may be excessive given this is an expires-in).

Happy to go with whatever your best judgement is on this, don't be bound by precedent.